js/main.js

```diff
@@ -3,9 +3,8 @@ $(document).ready(function() {
   // ------------------------------
   // Settings
   // ------------------------------
-  var apiUrl       = 'https://api.coinmarketcap.com/v1/ticker/?limit=0';
-  var iconsBaseUrl = 'https://rawgit.com/atomiclabs/cryptocurrency-icons/master';
-  var dataJson     = 'https://raw.githubusercontent.com/atomiclabs/cryptocurrency-icons/master/manifest.json';
+  var iconsBaseUrl = 'https://cdn.rawgit.com/atomiclabs/cryptocurrency-icons/master';
+  var dataJson     = 'https://cdn.rawgit.com/atomiclabs/cryptocurrency-icons/master/manifest.json';
```

Comment:
We can't use the CDN version of rawgit as it will not update when master updates.

Reply:
ok